lib/ansible/module_utils/netapp.py

```diff
@@ -80,12 +80,22 @@ def has_sf_sdk():
     return HAS_SF_SDK
 
 
-def ontap_sf_host_argument_spec():
+def na_ontap_host_argument_spec():
 
     return dict(
         hostname=dict(required=True, type='str'),
         username=dict(required=True, type='str', aliases=['user']),
         password=dict(required=True, type='str', aliases=['pass'], no_log=True),
+        https=dict(required=False, type='str', default='False')
+    )
+
+
+def ontap_sf_host_argument_spec():
```

Comment:
ontap_sf, or na_sf?

Reply:
this is the original function that works for the old CDOT, and SF. I'm setting this back to how it was before i had touched it. The function above (na_ontap_host_argument_spec) is the new one, for the new NA_ontap modules.